app/controllers/api/subcollections/authentications.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module Subcollections
+    module Authentications
+      def authentications_query_resource(object)
+        object.try(:authentications) || []
```

Comment:
would prefer:

```
object.respond_to?(:authentications) ? object.authentications : []
```

actual nil vs. [] is dealt differently in renderer. Thanks.